blocks/header/header.css

```diff
@@ -491,6 +491,83 @@ header .lang-picker > a::after {
   vertical-align: bottom;
 }
 
+/* stylelint-disable-next-line no-descending-specificity */
+header .website-picker > a::after {
```

Comment:
The styles look very similar to lang picker. Can we just reuse the same styles? The name can be generalised.